test/filters/asApplicationTypeConfirmationGoalId.test.ts

```diff
@@ -0,0 +1,28 @@
+import { assert } from 'chai'
+
+import { asApplicationTypeConfirmationGoalId } from 'app/filters/asApplicationTypeConfirmationGoalId.filter'
+import ApplicationType from 'app/models/dto/applicationType.enum'
+import PiwikConfig from 'app/models/piwikConfig'
+
+const piwikConfig: PiwikConfig = {
+  url: '',
+  siteId: '',
+  landingPageStartGoalId: 999,
+  confirmationPagePDFGoalId: 998,
+  limitedCompanyGoalId: 997,
+  partnershipGoalId: 996,
+  limitedCompanyConfirmationGoalId: 1000,
+  partnershipConfirmationGoalId: 1001,
+  multiDirectorConfirmationGoalId: 1002,
+  singleDirectorConfirmationGoalId: 1003
+  }
```

Comment:
please use a fixture for generating piwik config i.e. `piwik.fixtures.ts`. Closing bracket is indented wrong btw